app/decorators/models/submission_max_attempts_decorator.rb

```diff
@@ -2,20 +2,33 @@
   after_commit :send_max_attempts_alert
 
   def send_max_attempts_alert
-    return unless used_attempts && max_attempts
-    if used_attempts >= max_attempts
-      teachers_to_alert.each do |teacher|
-        AlertsService::Client.create(
-          AlertsService::Alerts::MaxAttemptsReached.new(
-            teacher_id: teacher.id, 
-            student_id: user.id, 
-            assignment_id: assignment.id
-          )
-        )
-      end
+    return unless student_locked?
+    teachers_to_alert.each do |teacher|
+      AlertsService::Client.create(
+        :max_attempts_reached,
+        teacher_id: teacher.id,
+        student_id: user.id,
+        assignment_id: assignment.id,
+        course_id: assignment.course.id,
+        score: score
+      )
     end
   end
-  
+
+  def student_locked?
+    return unless used_attempts && max_attempts
+    return unless used_attempts >= max_attempts
+
+    content_tag = ContentTag.find_by(content_id: assignment.id, content_type: 'Assignment')
+    return unless content_tag
+    context_module = content_tag.context_module
+    return unless context_module
+    requirement = context_module.completion_requirements.find { |req| req[:id] == content_tag.id }
+    return unless requirement
+    return unless requirement[:min_score]
```

Comment:
could turn this into a one-liner, not gonna block over it
return unless requirement && requirement[:min_score]